app/components/breadcrumb_nav_component.html.erb

```diff
@@ -1,14 +1,17 @@
 <nav class="breadcrumbs">
   <nav id="breadcrumbs" aria-label="breadcrumb" class="container-xxl">
     <ol class="breadcrumb">
-      <% breadcrumbs.each do |breadcrumb| %>
+      <% breadcrumbs.each do |breadcrumb|
+        full_title = breadcrumb[:title].presence || 'No title'
```

Comment:
Can you move these to a method in the class file?

Reply:
good idea, done